versioned_docs/version-1.0/language/resources.mdx

```diff
@@ -344,6 +285,46 @@ parent.child <-> otherChild
 // `otherChild` is the first child, Child 1.
 ```
 
+When a resource containing nested resources in fields is destroyed with a `destroy` statement,
+all the nested resources are also destroyed. 
```

Comment:
Maybe add back the example, remove the destructor from it, maybe modify it to support multiple children, and show/explain how the children are automatically destroyed when the parent is destroyed. 

Also, a reader might wonder if it is (still) possible to influence that behaviour. Only the "destroy events" section below mentioned that it isn't. Maybe add a description here in this section explicitly, and describe how the automatic destruction behaviour cannot be influenced, and it is not possible to run arbitrary code when a resource is destroyed